### PR TITLE
For #15497: Hide and remove the three dot menu from the Desktop Bookmarks folder

### PR DESCRIPTION
**Pull request by sijanr**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Review by Mugurell — CHANGES_REQUESTED** [review]
Thanks for looking into this!

This indeed fixes the issue (because of the `enabled = false` part) but using `hideAndDisable` would still have the button in the layout, just invisible. 

Should we use `removeAndDisable` so that the unactionable button won't be part of the layout?

**Review by sijanr — COMMENTED** [review]
This should remove the button as well, right?

**Comment by Mugurell** [comment]
> This should remove the button as well, right?

I think you had some specific code in mind that you wanted to point at...
I'm seeing that indeed, below the change needed here we use `hideAndDisable` which would still have that button occupy space.
And then if we are to use `removeAndDisable` for the "desktop bookmarks" node we would get a different behavior.
I'm not sure having the button still occupying space was intended or not. In the original ticket that implemented this I don't see exact specifications for this. 
I'll ask in the ticket for UX feedback. Thank you for pointing this out!

**Comment by Mugurell** [comment]
Seeing that previously we were setting `View.GONE` for folders and `View.INVISIBLE` for other nodes and Emily also recommending using `removeAndDisable` here let's merge this and file another ticket to discuss with UX if this is the expected behavior or not.

**Comment by Mugurell** [comment]
bors try

File: app/src/main/java/org/mozilla/fenix/library/bookmarks/viewholders/BookmarkNodeViewHolder.kt [hunk 1]
```diff
@@ -12,10 +12,7 @@ import mozilla.components.concept.storage.BookmarkNode
 import mozilla.components.concept.storage.BookmarkNodeType
 import mozilla.components.support.ktx.android.content.getDrawableWithTint
 import org.mozilla.fenix.R
-import org.mozilla.fenix.ext.components
-import org.mozilla.fenix.ext.hideAndDisable
-import org.mozilla.fenix.ext.loadIntoView
-import org.mozilla.fenix.ext.showAndEnable
+import org.mozilla.fenix.ext.*
```
Comment (Mugurell) [inline app/src/main/java/org/mozilla/fenix/library/bookmarks/viewholders/BookmarkNodeViewHolder.kt:15]: Wildcard imports will fail our detekt/ktlint.
Please use individual imports.

**Comment by Mugurell** [comment]
bors try

**Review by Mugurell — APPROVED** [review]
LGTM
Thank you for pushing through with this!

**Comment by Mugurell** [comment]
bors r=Mugurell

**Pull request merged**